examples/draw-brush.c

```diff
@@ -0,0 +1,125 @@
+
+#include <tchar.h>
+#include <windows.h>
+
+#include <wdl.h>
+
+
+static HWND hwndMain = NULL;
+
+
+static const WD_COLOR drawColors[3] =
+        { WD_RGB(255,0,0), WD_RGB(0,255,0), WD_RGB(0,0,255) };
+static const WD_COLOR fillColors[3] =
+        { WD_ARGB(63,255,0,0), WD_ARGB(63,0,255,0), WD_ARGB(63,0,0,255) };
+
+static void
+MainWinPaintToCanvas(WD_HCANVAS hCanvas)
+{
+    WD_HBRUSH hBrush;
+    int i;
+
+    wdBeginPaint(hCanvas);
+    wdClear(hCanvas, WD_RGB(255,255,255));
+    wdBeginPaint(hCanvas);
+    wdClear(hCanvas, WD_RGB(255,255,255));
+
+    for(i = 0; i < 3; i++) {
+        float x = 10.0f + i * 20.0f;
+        float y = 10.0f + i * 20.0f;
+
+        WD_POINT p0;
+        p0.x = x;
+        p0.y = y;
+        WD_POINT p1;
+        p1.x = x + 100.0f;
+        p1.y = y + 100.0f;
+        float offsets[3] = {0, 0.5f, 1.0f};
```

Comment:
The code in examples should be easily readable for newcomers.

Please declare the `offsets[]` and `colors[]` at the same place. They are related. Rename `drawColors` to something else. It's misleading name. If you make it local variable, `colors[]` would suffice. If global, `brushColors[]` would be likely better. Either way `offsets[]` should have same prefix.